File: DESAFIOS-SEMANA01/desafio01.cpp
//https://leetcode.com/problems/shuffle-the-array/
//desafio 01

class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> aux;
        for(int i = 0; i < (nums.size())/2; i++){
            aux.push_back(nums[i]);
            aux.push_back(nums[i+n]);
        }
        return aux;
    }
    
};

File: DESAFIOS-SEMANA01/desafio01.go
//go
//Desafio 01 em Go
//https://leetcode.com/problems/find-numbers-with-even-number-of-digits/

package main

func findNumbers(nums []int) int {
    var final []int
    for _, i := range(nums){
        aux :=  len(strconv.Itoa(i))%2
        if aux == 0 {
            final= append(final, i)
        }
    }
    return len(final)
    
}
